keycloak-rest-api-extensions-tests/src/test/java/io/cloudtrust/keycloak/services/api/admin/CtAdminRootTest.java

```diff
@@ -1,58 +1,59 @@
 package io.cloudtrust.keycloak.services.api.admin;
 
 import com.fasterxml.jackson.core.type.TypeReference;
-import com.fasterxml.jackson.databind.ObjectMapper;
 import io.cloudtrust.keycloak.AbstractRestApiExtensionTest;
+import io.cloudtrust.keycloak.config.ServerConfig;
 import io.cloudtrust.keycloak.representations.idm.DeletableUserRepresentation;
 import io.cloudtrust.keycloak.services.resource.api.model.EmailInfo;
-import io.cloudtrust.keycloak.test.container.KeycloakDeploy;
-import org.apache.http.NameValuePair;
-import org.apache.http.message.BasicNameValuePair;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
-
 import java.io.IOException;
 import java.net.URISyntaxException;
 import java.time.Duration;
 import java.util.Collections;
 import java.util.List;
+import org.apache.http.NameValuePair;
+import org.apache.http.message.BasicNameValuePair;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.keycloak.testframework.annotations.KeycloakIntegrationTest;
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.is;
 
-@ExtendWith(KeycloakDeploy.class)
+@KeycloakIntegrationTest(config = ServerConfig.class)
 class CtAdminRootTest extends AbstractRestApiExtensionTest {
-    private static final TypeReference<List<DeletableUserRepresentation>> deletableUserListType = new TypeReference<List<DeletableUserRepresentation>>() {
-    };
-    private static final TypeReference<List<EmailInfo>> emailInfoListType = new TypeReference<>() {
-    };
+
```

Comment:
Avoid adding empty lines... If you like them and someone else does not, git history will be full of add/del/add/del of empty lines. (other example: we don't need empty lines after the first opening curly bracket of the class and before the last one)